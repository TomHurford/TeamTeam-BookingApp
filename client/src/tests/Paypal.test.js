import React from 'react';
import renderer from 'react-test-renderer';
import Paypal from '../components/Paypal';

test('Paypal snapshot test', () => {
  const component = renderer.create(<Paypal />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});