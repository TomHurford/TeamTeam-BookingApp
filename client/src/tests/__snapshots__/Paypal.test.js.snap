// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paypal snapshot test 1`] = `
<div
  className="page-container"
>
  <div
    className="underlay"
  />
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "justifyContent": "center",
      }
    }
  >
    <div
      style={
        Object {
          "maxWidth": "500px",
        }
      }
    />
  </div>
</div>
`;
